packages/components/src/components/NFT/NFT.tsx

```diff
@@ -54,8 +64,12 @@ export const NFT = ({ contractAddress, tokenId }: NFTProps) => {
           animationUrl: data.animation_url,
         });
       }
-    } catch (error: any) {
-      setErrorMessage(error.message);
+    } catch (error) {
+      if (error instanceof Error) {
```

Comment:
Could be good to create a helper for this as we're handling errors in quite a few places. What do you reckon? Can add a separate ticket for this.

Reply:
Makes sense. It seems like we're handling errors (and the typing of them) in a couple ways. I'll make an issue to figure out the best way to handle errors in our code, and use whatever helper function i make across the codebase

Reply:
https://github.com/Developer-DAO/web3-ui/issues/133

Reply:
@maximebonhomme feel free to create that ticket. Good suggestion